app/views/workbooks/_workbook_form.html.haml

```diff
@@ -0,0 +1,27 @@
+= form_for(@workbook, html: { multipart: true }) do |f|
+  = render 'shared/error_messages', object: f.object
+  .form-group
+    = f.label :name, class: "control-label"
+    = f.text_field :name, class: "form-control", placeholder: "Enter workbook name"
+  .form-group
+    = f.label :subject_id, "Select subject", class: "control-label"
+    = f.collection_select(:subject_id, Subject.all, :id, :name, {}, class: "form-control")
+  .form-group
+    = f.label :description, class: "control-label"
+    = f.text_area :description, class: "text-wysiwyg form-control", rows: 15
+
+  .form-group
+    = f.label :attachment, class: "control-label"
+    - if @workbook.persisted?
+      %p
+        Current file:
+        = link_to File.basename(@workbook.attachment.path), @workbook.attachment.url
+        %span.help-block  If you upload another file the old one will be deleted.
+    = f.file_field :attachment, accept: 'doc/zip/pdf'
```

Comment:
yes, that's why it should be extracted now. So that later, when you get the full list, you touch only the constant.